programs-remote-tests/tests/_support/AcceptanceHelper.php

```diff
@@ -12,7 +12,7 @@ public function changeDefaultPages(\AcceptanceTester $I, $programPage, $teacherP
 
         // Save settings
         $I->amOnPage('/wp-admin/admin.php?page=options-mbm');
-        $I->fillField('#rs_domain', 'tests');
+        $I->fillField('#rs_domain', 'tests.qa0');
```

Comment:
does tests.secure.retreat.guru not work anymore?

Reply:
see env comment above